### PR TITLE
i18n(es): Translated missing error pages.

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.

Translated to spanish missing error pages as follows:
  - `image-missing-alt.mdx`
  - `invalid-image-service.mdx`
  - `missing-image-dimension.mdx`
  - `unsupported-image-formate.mdx`

### CODE REVIEW

File: src/content/docs/es/reference/errors/image-missing-alt.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Do not make edits to it directly, they will be overwritten.
+# Instead, change this file: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+# Translators, please remove this note and the <DontEditWarning/> component.
+
+title: Missing alt property
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **ImageMissingAlt**: La propiedad alt es requeridad. (E03022)
```
Comment (agustinmulet) [inline src/content/docs/es/reference/errors/image-missing-alt.mdx:12]:
```suggestion
> **ImageMissingAlt**: La propiedad alt es requerida. (E03022)
```

File: src/content/docs/es/reference/errors/image-missing-alt.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## ¿Qué salió mal?
+La propiedad `alt` te permite proporcionar texto alternativo descriptivo a los usuarios de lectores de pantalla y otras tecnologías de asistencia. Para garantizar que tus imágenes sean accesibles, el componente `Image` requiere que se especifique un `alt`.
+
+Si la imagen es meramente decorativa (es decir, no contribuye a la comprensión de la página), establezca `alt=""` para que los lectores de pantalla sepan ignorar la imagen.
```
Comment (agustinmulet) [inline src/content/docs/es/reference/errors/image-missing-alt.mdx:17]:
```suggestion
Si la imagen es meramente decorativa (es decir, no contribuye a la comprensión de la página), establece `alt=""` para que los lectores de pantalla sepan ignorar la imagen.
```

File: src/content/docs/es/reference/errors/invalid-image-service.mdx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# Do not make edits to it directly, they will be overwritten.
+# Instead, change this file: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+# Translators, please remove this note and the <DontEditWarning/> component.
+
+title: Error while loading image service
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **InvalidImageService**: Hubo un error cargando el servicio configurado de imagen. Porfavor revisa el stack trace para más información. (E03023)
```
Comment (agustinmulet) [inline src/content/docs/es/reference/errors/invalid-image-service.mdx:12]:
```suggestion
> **InvalidImageService**: Hubo un error cargando el servicio configurado de imagen. Por favor revisa el stack trace para más información. (E03023)
```

File: src/content/docs/es/reference/errors/invalid-image-service.mdx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## ¿Qué salió mal?
+Hubo un error mientras cargaba el servicio de imagen configurado. Esto puede ser causado por varios factores, como que tu servicio de imagen no exporte correctamente un objeto compatible en su exportación por defecto o una ruta incorrecta.
+
+Si crees que tu servicio está configurado correctamente y este error es incorrecto, porfavor [abre un issue](https://astro.build/issues/).
```
Comment (agustinmulet) [inline src/content/docs/es/reference/errors/invalid-image-service.mdx:17]:
```suggestion
Si crees que tu servicio está configurado correctamente y este error es incorrecto, por favor [abre un issue](https://astro.build/issues/).
```

File: src/content/docs/es/reference/errors/missing-image-dimension.mdx [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> Falta el atributo de ancho y alto para `IMAGE_URL`. Cuando se usan imágenes remotas, siempre se requieren ambas dimensiones para evitar el desplazamiento acumulativo de la capa (CLS). (E03024)
+
+## ¿Qué salió mal?
+Para imágenes remotas, no se pueden inferir `width` y `height` del archivo original. Por lo tanto, para evitar el CLS, siempre se requieren esas dos propiedades.
+
+Si tu imagen está dentro de tu carpeta `src`, probablemente quieras importarla en su lugar. Consulte [la guía de importaciones para obtener más información](/es/guides/imports/#otros-recursos).
```
Comment (agustinmulet) [inline src/content/docs/es/reference/errors/missing-image-dimension.mdx:17]:
```suggestion
Si tu imagen está dentro de tu carpeta `src`, probablemente quieras importarla en su lugar. Consulta [la guía de importaciones para obtener más información](/es/guides/imports/#otros-recursos).
```

File: src/content/docs/es/reference/errors/unsupported-image-format.mdx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **UnsupportedImageFormat**: Recibí un formato no admitido `FORMAT` de `IMAGE_PATH`. Actualmente solo se admiten SUPPORTED_FORMATS.JOIN(', ') para optimización. (E03025)
+
+## ¿Qué salió mal?
+Los servicios precargados de imagen no tienen soporte para optimizar todos los formatos de imagen.
+
+Para formatos no admitidos como SVG y GIF, puede usar una etiqueta `img` directamente:
```
Comment (agustinmulet) [inline src/content/docs/es/reference/errors/unsupported-image-format.mdx:17]:
```suggestion
Para formatos no admitidos como SVG y GIF, puedes usar una etiqueta `img` directamente:
```

**Review by agustinmulet — CHANGES_REQUESTED** [review]
Good job! Left a couple of suggestions 🙌

**Comment by dreyfus92** [comment]
Grande @agustinmulet, gracias por el review 😄

**Comment by Yan-Thomas** [comment]
Great job @dreyfus92 & @agustinmulet, just a thing: we also need to update the `error-reference.mdx` page here so the new errors are listed on it, could you add this to the PR? Thanks 🙌

**Comment by dreyfus92** [comment]
I will handle it in a bit @Yan-Thomas.

**Review by Yan-Thomas — APPROVED** [review]

**Pull request merged**